resources/server_install.rb

```diff
@@ -30,6 +30,7 @@
 action :install do
   node.run_state['postgresql'] ||= {}
   node.run_state['postgresql']['version'] = new_resource.version
+  puts "HELLO DAN THE VERSION IS #{new_resource.version}"
```

Comment:
?

Reply:
I've removed it from latest, but I was debugging, some version numbers were... Wrong.